### PR TITLE
Add title tooltip to icons

### PR DESCRIPTION
**Pull request by ThomDietrich**
2016-07-21 23_35_10-start

### CODE REVIEW

File: extensions/ui/iconset/org.eclipse.smarthome.ui.iconset.classic/README.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ for icon in icons/*.png; do
   name="$(basename "${icon}" | cut -d '.' -f1)"
 
   if [ "${name}" != 'none' ]; then
-    echo "![${name}](${icon})" >> "${README}"
+    echo "![${name}](${icon} \"Icon name: ${name}\")" >> "${README}"
```
Comment (watou) [inline extensions/ui/iconset/org.eclipse.smarthome.ui.iconset.classic/README.sh]: Is "Icon name: " necessary?  Would ${name} be enough?

Comment (ThomDietrich) [inline extensions/ui/iconset/org.eclipse.smarthome.ui.iconset.classic/README.sh]: Of course ;)

**Comment by kaikreuzer** [comment]
That's really useful, I thought of that as well.
@ThomDietrich Please note that you need to properly sign-off your commits when contributing, see https://github.com/eclipse/smarthome/blob/master/CONTRIBUTING.md#contributing-to-eclipse-smarthome.

**Comment by ThomDietrich** [comment]
Didn't think that applies for a patch that small that I created it directly on github ;)
The last commit is signed-off.

**Comment by kaikreuzer** [comment]
Uh, wow, first time I see a gpg signed commit by someone :-)
Unfortunately, this still makes the Eclipse IP validation fail, because it requires the line "Signed-off-by: my@email.com" in the commit message - and it checks that there is a valid CLA for the provided email address. Adding this line in the commit message is also possible when doing changes through the Github UI, so no need to revert to a git tool.

**Comment by kaikreuzer** [comment]
Thanks - now the last remaining issue is that https://projects.eclipse.org/user/cla/validate says:

screen shot 2016-07-22 at 18 04 30

**Comment by ThomDietrich** [comment]
Wow these have been the most time consuming 11 characters of my life ^^ gpg signing is something I do with mail. When you said sign-off that's what came to mind first as the other one is the weaker proof (`-s` vs `-S`). Sorry for not studying the contribution rules earlier.
Everything is signed (gpg and in text) and there should be a valid CLA now...

**Comment by kaikreuzer** [comment]
> Wow these have been the most time consuming 11 characters of my life

Haha, yeah, sorry for these requirements, but it is something we have to follow at the Eclipse Foundation. See it as a one-time time investment for all future contributions you do - hopefully more than 11 chars! Thanks :-)

**Pull request merged**